packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -130,7 +130,7 @@ export class CeramicCliUtils {
         config.httpApi.hostname = hostname
       }
       if (port) {
-        config.httpApi.port = port
+        config.httpApi.port = _validatePort(port)
```

Comment:
do we even need this here if we're going to check it again in the CeramicDaemon constructor?